tests/test_views.py

```diff
@@ -0,0 +1,129 @@
+import sys
+from unittest import skipIf, skipUnless
+
+from django import VERSION as DJANGO_VERSION
+from django.core.urlresolvers import clear_url_caches
+
+from cms.api import add_plugin, create_page
+from cms.appresolver import clear_app_resolvers
+from cms.test_utils.testcases import CMSTestCase
+
+
+DJANGO_111 = DJANGO_VERSION[:2] >= (1, 11)
+
+
+class SubmitFormViewTest(CMSTestCase):
+
+    def setUp(self):
+        self.APP_MODULE = 'aldryn_forms.cms_apps.FormsApp'
```

Comment:
why not call reload_urls method?

Reply:
https://github.com/aldryn/aldryn-forms/pull/161/files/da5a101bbad04906bc05db25bafa96a55e065c66#diff-a4f5c4d64d4811b15e1405573c00f3f4R52